package-lock.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "web-world-wind",
-  "version": "0.0.1",
+  "name": "@nasaworldwind/webww",
+  "version": "0.1.2",
```

Comment:
The project name and version in package-lock.json are incorrect. Is this file out of date?